tipocket-ctl/tpctl/scripts/env_raw.sh

```diff
@@ -0,0 +1,72 @@
+# WARNING: All commands connect to `tidb` container of tidb pods by default.
```

Comment:
We should add `scripts/env_raw.sh` as tpctl package data.
https://packaging.python.org/guides/distributing-packages-using-setuptools/#package-data

Reply:
You are right.